runners/direct-java/pom.xml

```diff
@@ -310,6 +318,13 @@
 
     <!-- test dependencies -->
     <dependency>
+      <groupId>org.apache.beam</groupId>
```

Comment:
This is an actual dependency to pull in `WindowMatchers`.

Note that it is using `classifier` instead of `type`. I suspect that `classifiers` is almost always better than `test-jar` since it is more abstract, but that is more Maven work than I'd like to do right now.